flood_adapt/adapter/sfincs_adapter.py

```diff
@@ -934,6 +939,14 @@ def _add_forcing_wind(
                 raise ValueError("No path to rainfall track file provided.")
             # data already in metric units so no conversion needed
             self._add_forcing_spw(wind.path)
+        elif isinstance(wind, WindNetCDF):
+            ds = wind.read()
+            # TODO timeframe
```

Comment:
not sure what this TODO refers to, can you either resolve it, or make it more descriptive please?